workflow_array_ephys/plotting/plot_psth.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def _plot_spike_raster(aligned_spikes, trial_ids=None, vlines=[0], ax=None, title='',
+                       xlim=None):
+    if not ax:
+        fig, ax = plt.subplots(1, 1)
+
+    raster = np.concatenate(aligned_spikes)
+    if trial_ids is None:
+        trial_ids = range(len(aligned_spikes))
+
+    trial_ids = np.concatenate([[t] * len(s)
+                                for t, s in zip(trial_ids, aligned_spikes)]).astype(int)
+
+    assert len(raster) == len(trial_ids)
+
+    ax.plot(raster, trial_ids, 'r.', markersize=1)
+
+    for x in vlines:
+        ax.axvline(x=x, linestyle='--', color='k')
+
+    if xlim:
+        ax.set_xlim(xlim)
+    ax.set_axis_off()
+    ax.set_title(title)
+
+
+def _plot_psth(psth, psth_edges, vlines=[0], ax=None, title='', xlim=None):
+    if not ax:
+        fig, ax = plt.subplots(1, 1)
+
+    ax.plot(psth_edges, psth, 'r')
```

Comment:
Should we plot a histogram instead?

Reply:
Sorry - can you be more specific? Currently we have the peri-stim histogram in the notebooks. Were you thinking of another histogram setup?

Reply:
I did a deep dive here. The reason it looks off is because we separate out the pieces:

1. [Generate histogram](https://github.com/datajoint/workflow-array-ephys/blob/event/workflow_array_ephys/analysis.py#L85-L86)
2. [Store as longblob](https://github.com/datajoint/workflow-array-ephys/blob/event/workflow_array_ephys/analysis.py#L94)
3. [Render as line graph](https://github.com/datajoint/workflow-array-ephys/blob/event/workflow_array_ephys/plotting/plot_psth.py#L35)

Under normal circumstances (i.e., not sparse fake data), this would look like a the top silhouette of a histogram, without delineation between boxes. Because our bins are so small (.04), we only find histograms of 1 item, and we're so zoomed out that it looks like a spike. 

Should we...
1. Revise with a coarser bin size default that won't apply as well to real data?
2. Adjust the notebook to have a larger bin size? (Note: this has some complications with needing to remove `bin_size` from the dictionary before populating the part table)
3. Kick the can down the road to viable MVP data for this demo, replacing the fake trials?

@ttngu207 - What do you think?

Reply:
Thanks!